examples/manifest.toml

```diff
@@ -27,84 +27,85 @@ examples = [
 order = 2
 title = "Robotics"
 prelude = """
-Robitics examples.
+Examples related to robotics, autonomous systems, and interfacing with sensor hardware.
 """
-examples = ["nuscenes", "ros_node", "rrt-star"]
+examples = [
+  "ros_node",
+  "nuscenes",
+  "live_depth_sensor",
+  "lidar",
+  "rrt-star",
+  "rgbd",
+  "nv12",
```

Comment:
Unless we have a strong reason not to, I suggest we sort these lists lexicographically

Reply:
That is order in which the examples are displayed on the web page, so the order should be deliberate, not lexicographic. I did try to make a pass on that order, putting the nicer/more relevant/etc. one in front.

Reply:
👍 Then also add a comment here to then to explain that motivation to the `manifest.toml` reader

Reply:
👍🏻